packages/apps/plugins/plugin-chess/src/ChessPlugin.tsx

```diff
@@ -60,36 +61,41 @@ export const ChessPlugin = (): PluginDefinition<ChessPluginProvides> => {
                   dispatch,
                 }),
               );
+            });
 
-              // Add all games to the graph.
-              const query = space.db.query(Filter.schema(GameType));
-              subscriptions.add(query.subscribe());
-              let previousObjects: GameType[] = [];
-              subscriptions.add(
-                effect(() => {
-                  const removedObjects = previousObjects.filter((object) => !query.objects.includes(object));
-                  previousObjects = query.objects;
+            client.spaces
+              .get()
+              .filter((space) => !!enabled.find((key) => key.equals(space.key)))
+              .forEach((space) => {
```

Comment:
Since we're repeating this in every plugin, do you think it's worth having a utility?

Reply:
hoping it's temporary :)